src/pybind/mgr/dashboard/controllers/rbd.py

```diff
@@ -471,13 +471,24 @@ def __init__(self):
         super().__init__()
         self.rbd_inst = rbd.RBD()
 
+    def _validate_namespace(self, ioctx, namespace):
+        namespaces = self.rbd_inst.namespace_list(ioctx)
+        if namespace and namespace not in namespaces:
+            raise DashboardException(
+                msg='Namespace not found',
+                code='namespace_not_found',
+                component='rbd')
```

Comment:
should we move it inside the `services/rbd.py` directory? specifically inside the [RbdService class](https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/services/rbd.py#L268), to keep the controller clean?